.github/workflows/build.yml

```diff
@@ -1,28 +1,35 @@
-name: Build
-on: [push, pull_request]
+name: Build Code
+
+on:
+  push:
+  pull_request:
+    types: [opened]
+
+env:
+  INEXOR_VULKAN_VERSION: "1.3.236.0"
```

Comment:
```suggestion
  INEXOR_VULKAN_VERSION: "1.3.215.0"
```

Reply:
I think now I should have fixed it in every place, I searched through everything with VS code. Done.